apps/payments/next/.storybook/preview.ts

```diff
@@ -0,0 +1,13 @@
+import '@fxa/shared/styles';
+// import { withThemeByDataAttribute } from '@storybook/addon-styling';
```

Comment:
I thought about that but Bianca's PR left areas commented out so I thought this would benefit us to have it rather than remove it. I feel like this will get overlooked and put to the back-burner but we will be adding in a11y to M1, no?

Reply:
My 2 cents on this, ideally we wouldn't ship with commented out code. I quite like the approach of leaving a `//TODO` comment referencing a specific Jira ticket that will complete the work. 

This was recommended to me a few weeks ago, and quite liked it. This way the code stays clean, and we don't loose track of work we need to pick up in future.